packages/messenger-widget/src/hooks/auth/useAuth.ts

```diff
@@ -4,6 +4,7 @@ import {
     normalizeEnsName,
 } from '@dm3-org/dm3-lib-profile';
 import { useContext, useEffect, useMemo, useState } from 'react';
+// @ts-ignore
```

Comment:
what is the reason for the ts-ignore at the wagmi imports?

Reply:
As per the wagmi documentation, a declaration for import wagmi needs to be created in the latest version. But when that declaration is made the imports from wagmi gives error. So its set to typescript ignored. But it works on execution.